src/App.js

```diff
@@ -94,6 +104,20 @@ function App({ data }) {
               />
             ))}
           </Route>
+          <Route exact path="/kritzelkopf">
+            <Container>
+              <Profile />
+              <Welcome />
+              <Interests />
+              <Skills />
+              <Facts />
+              <Portfolio />
+              <Xing />
+              <Github />
+              <LinkedIn />
+              <BlogText />
+            </Container>
```

Comment:
I think you should extract this Container into an own Component like 'Kritzelkopf'
Makes it easier to read and you then have for every page it's own component.